src/components/molecules/Header/Header.tsx

```diff
@@ -1,11 +1,16 @@
-import styles from './header.module.scss';
-import concatClassNames from '@/src/app/utils/concatClassNames';
-import content from '@/src/app/content/content';
 import Link from 'next/link';
+import styles from './header.module.scss';
```

Comment:
this failed to run locally for me because it should be `Header.module.scss`

Reply:
Fixed